rosdep/python.yaml

```diff
@@ -7552,6 +7552,15 @@ python3-openhsi-pip:
   ubuntu:
     pip:
       packages: [openhsi]
+python3-openpyxl:
+  alpine: [py3-openpyxl]
+  arch: [python-openpyxl]
+  debian: [python3-openpyxl]
+  fedora: [python3-openpyxl]
+  gentoo: [dev-python/openpyxl]
+  opensuse: [python3-openpyxl]
+  rhel: ['python%{python3_pkgversion}-openpyxl']  
```

Comment:
> It seems like there's a RHEL package that could be added.

Thanks for catching. Added RHEL.

Since I see some entries for RHEL uses this `%{python3_pkgversion}` notation, I followed that without understanding what for it is needed (and also not understanding why non-RHEL entries don't use that).